### PR TITLE
Added Downsampled ImageNet dataset.

### PR DESCRIPTION
**Pull request by ChanchalKumarMaji**
Related issue #45.

### CODE REVIEW

**Comment by us** [comment]
@ChanchalKumarMaji you can squash commits that's have unnecessary commit messages like merging commit, faulty commit etc.

File: tensorflow_datasets/image/downsampled_imagenet.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+from __future__ import print_function
+
+import os
+
+import tensorflow as tf
+
+from tensorflow_datasets.core import api_utils
+import tensorflow_datasets.public_api as tfds
+
+# From https://arxiv.org/abs/1601.06759
```
Comment (us) [inline tensorflow_datasets/image/downsampled_imagenet.py]: I think it's unnecessary.

Comment (ChanchalKumarMaji) [reply tensorflow_datasets/image/downsampled_imagenet.py]: Actually I followed [this](https://github.com/tensorflow/datasets/blob/90c0b38438afcb56044f13b11a426d013d0c8848/tensorflow_datasets/image/cycle_gan.py#L29).

Comment (us) [reply tensorflow_datasets/image/downsampled_imagenet.py]: Already written [there](https://github.com/tensorflow/datasets/blob/90c0b38438afcb56044f13b11a426d013d0c8848/tensorflow_datasets/image/cycle_gan.py#L41) so it be written twice. It's also unnecessary.

File: tensorflow_datasets/image/downsampled_imagenet.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+_CITATION = """\
+@article{DBLP:journals/corr/OordKK16,
+  author    = {A{\"{a}}ron van den Oord and
+               Nal Kalchbrenner and
+               Koray Kavukcuoglu},
+  title     = {Pixel Recurrent Neural Networks},
+  journal   = {CoRR},
+  volume    = {abs/1601.06759},
+  year      = {2016},
+  url       = {http://arxiv.org/abs/1601.06759},
```
Comment (us) [inline tensorflow_datasets/image/downsampled_imagenet.py]: url already written in there

File: tensorflow_datasets/image/downsampled_imagenet.py (hunk 1 above)
Comment (ChanchalKumarMaji) [reply tensorflow_datasets/image/downsampled_imagenet.py]: Okk, I see.

File: tensorflow_datasets/image/downsampled_imagenet.py (hunk 2 above)
Comment (ChanchalKumarMaji) [reply tensorflow_datasets/image/downsampled_imagenet.py]: Done.

**Pull request merged**